go.mod

```diff
@@ -40,3 +40,5 @@ require (
 	knative.dev/pkg v0.0.0-20210510175900-4564797bf3b7
 	knative.dev/reconciler-test v0.0.0-20210512062647-d5adf72546be
 )
+
+replace knative.dev/eventing => github.com/cardil/eventing v0.11.1-0.20210518185315-14a9b5d91b31
```

Comment:
/hold

This PR uses integration branch of https://github.com/cardil/eventing/tree/integration/eventing-sandbox%2Feventing-kafka%2Fpr-563 which merges knative/eventing#4815 and knative/eventing#5321.

Do not merge this PR, until those PRs are merged to eventing and this code is restored.

Reply:
knative/eventing#4815 and knative/eventing#5321 are merged and integration branch is removed.

/unhold